sdcm/keystore.py

```diff
@@ -77,7 +77,7 @@ def get_ec2_ssh_key_pair(self):
         return self.get_ssh_key_pair(name="scylla_test_id_ed25519")
 
     def get_gce_ssh_key_pair(self):
-        return self.get_ssh_key_pair(name="scylla-test")
+        return self.get_ssh_key_pair(name="scylla_test_id_ed25519")
```

Comment:
Is it really correct that GCE gets that new `scylla_test_id_ed25519` key?

Reply:
yes, starting with this change it would